accepted/future-releases/class-modifiers/feature-specification.md

```diff
@@ -525,37 +533,44 @@ mixinModifier     ::= 'sealed' | 'base' | 'interface' | 'final'
 
 It is a compile-time error to:
 
-*   Extend a class marked `interface` or `final` outside of the library where it
-    is declared.
-
-*   Implement a class or mixin marked `base` or `final` outside of the library
+*   Extend a class marked `interface`, `final` or `sealed` outside of the library
     where it is declared.
-
-*   Mix in a mixin marked `interface` or `final` outside of the library where it
-    is declared.
-
+*   Implement the interface of a class or mixin marked `base`, `final` or `sealed`
+    outside of the library where it is declared.
+*   Mix in a mixin or mixin class marked `interface`, `final` or `sealed` outside 
+    of the library where it is declared.
 *   Extend a class marked `base` outside of the library where it is declared
     unless the extending class is marked `base` or `final`. *This ensures that a
     subtype can't escape the `base` restriction of its supertype by offering its
     _own_ interface that could then be implemented without inheriting the
-    concrete implementation from the supertype.*
-
+    concrete implementation from the supertype.* <!-- Needs to account for `sealed` -->
 *   Mix in a mixin or mixin class marked `base` outside of the library where it
     is declared unless the class mixing it in is marked `base` or `final`. *As
     with the previous rule, ensures you can't get a backdoor interface on a
     mixin that doesn't want to expose one.*
-
 *   Mix in a class not marked `mixin` outside of the library where it is
     declared, unless the class declaration being used as a mixin is in a library
     whose language version is older than the version this feature ships in.
-
+*   Mix in a class not marked `mixin` inside the library where it is declared,
+    if the class has a superclass other than `Object` or the class declares a 
+    non-trivial generative constructor. If the library has a language version
+    older than the version this feature ships int, the class must not declare
+    any generative constructor.
```

Comment:
I think this is mostly covered by the bullet point on line 574 and should be merged with that.